src/api/app/views/webui/users/tasks/index.html.haml

```diff
@@ -1,85 +1,78 @@
-            = image_tag('ajax-loader.gif', class: 'result_spinner hidden')
-        %li
-          %a#all-requests-tab{ href: "#all-requests", title: "All Requests from #{User.session!.login}" } All Requests
-          = sprite_tag('reload', title: 'Reload', class: 'result_reload hidden', data: { table: 'all_requests_table' })
-          = image_tag('ajax-loader.gif', class: 'result_spinner hidden')
-    .tab#requests-in
-      = render(partial: 'webui/shared/requests_table', locals: { id: 'requests_in_table', source_url: user_requests_path(User.session!) })
-    .tab#requests-out
-      = render(partial: 'webui/shared/requests_table', locals: { id: 'requests_out_table', source_url: user_requests_path(User.session!) })
-    .tab#requests-declined
-      = render(partial: 'webui/shared/requests_table', locals: { id: 'requests_declined_table', source_url: user_requests_path(User.session!) })
-    .tab#all-requests
-      = render(partial: 'webui/shared/requests_table', locals: { id: 'all_requests_table', source_url: user_requests_path(User.session!) })
+.card#reviews
+  .bg-light
+    %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+      %li.nav-item
+        - cache "#{User.session!.cache_key}-reviews-in" do
+          %a.nav-link.text-nowrap.active{ href: '#reviews-in', title: "Requests that #{User.session!} has to review" }
+            Incoming Reviews
+            %span.badge.badge-primary
+              = User.session!.involved_reviews.count
 
-- if User.session!.involved_patchinfos.present?
-  .grid_16.alpha.omega.box.box-shadow
-    #patchinfos
-      .box-header.header-tabs
-        %ul
-          %li
-            %a{ href: "#patchinfos-in", title: "Requests that #{User.session!.login} has to merge" } Maintenance Requests
-      .tab#patchinfos-in
-        %table.compact#open-patchinfos-table{ title: "Running Maintenance Updates" }
-          %thead
-            %tr
-              %th Project
-              %th Package
-              %th Issues
-              %th{ style: 'width: 1%' } Actions
-          %tbody
-            - User.session!.involved_patchinfos.each do |pi|
-              %tr
-                - shortened_prj, shortened_pkg = elide_two(pi[:package][:project], pi[:package][:name], 60)
-                %td= link_to(shortened_prj, project_show_path(pi[:package][:project]))
-                %td= link_to(shortened_pkg, package_show_path(pi[:package][:project], pi[:package][:name]))
-                %td
-                  - pi[:issues].each do |i|
-                    = link_to i[:label], i[:url], title: i[:summary]
-                    ,
-                %td= link_to(sprite_tag('monitor', title: 'Monitor'), project_monitor_path(pi[:package][:project], pkgname: pi[:package][:name]))
+  .card-body
+    .tab-content#reviews-in
+      = render(partial: 'webui2/shared/requests_table', locals: { id: 'reviews_in_table', source_url: user_requests_path(User.session!),
```

Comment:
Please, use the partials that are into `webui/shared`

Reply:
done

Reply:
This one still needs to be changed.

Reply:
done!